src/components/Icon/Icon.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Icon = () => {
+  return (
```

Comment:
Since there's no logic here, return the value directly like this
`const Icon = () => <svg>..... </svg>`

Also change the name from icon, to scoped and meaningful name such as
DownloadIcon, RemoveCartIcon, CartDownloadIcon